src/main/resources/hudson/plugins/sshslaves/SSHLauncher/help_de.properties

```
@@ -22,6 +22,6 @@
 
 blurb=Startet einen Agent durch Kommandos, die \u00FCber eine gesicherte SSH-Verbindung \
 	gesendet werden. \
-	Der Agent muss dazu vom Master aus erreichbar sein, und Sie m\u00FCssen einen \
+	Der Agent muss dazu vom Regler aus erreichbar sein, und Sie m\u00FCssen einen \
```

If someone in the community could please tell me if this is the proper word to use

---

@daniel-beck @orrc

---

```suggestion
	Der Agent muss dazu vom Controller aus erreichbar sein, und Sie m\u00FCssen einen \
```

The doc was wrong, bad copy & paste